File: Backend/src/main/java/com/feedle/feedleapi/Networking/GetUsersResponse.java
package com.feedle.feedleapi.Networking;

import com.feedle.feedleapi.Models.User;

import java.util.List;

public class GetUsersResponse extends Request {
    private List<User> users;
    public GetUsersResponse(List<User> users)
    {
        super(RequestType.GetUsers);
        this.users = users;
    }

    public List<User> getUsers() {
        return users;
    }
}

File: FeedleDataTier/FeedleDataTier/Network/GetUsersResponse.cs
using System.Collections.Generic;
using System.Text.Json.Serialization;
using FeedleDataTier.Models;

namespace FeedleDataTier.Network
{
    public class GetUsersResponse : Request
    {
        [JsonPropertyName("users")] 
        public List<User> Users { get; set; }

        public GetUsersResponse(List<User> users) : base(RequestType.GetUsers)
        {
            this.Users = users;
        }
    }
}